### PR TITLE
javascript1-week2

### PR DESCRIPTION
**Pull request by baharehsichani**
finished my homework

### CODE REVIEW

File: javascript/javascript1/week2/index2.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        
+    }else 
+       return("you need to write youe name and your family")       
+}
+
+
+console.log(getFullname("","Hughes",true,"male"))
+console.log(getFullname("Benjamin","Hughes",true,"male"))
+console.log(getFullname("Benjamin","Hughes",false,"male"))
+console.log(getFullname("Bahareh","Sichani",false,"female"))
```
Comment (esteficodes) [inline javascript/javascript1/week2/index2.js:26]: Good solution! 👍

File: javascript/javascript1/week2/index3.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+    const result=day+(num%7)   
+
+    if(result<=7)
+      return (arrayOfWeekdays[result])
+     else
+      return (arrayOfWeekdays[(result%7)])  
+}
+
+console.log(getEventWeekday(3))
```
Comment (esteficodes) [inline javascript/javascript1/week2/index3.js:21]: Nice usage of the **remainder operator.** 💯 Don't forget that **var** is a little outdated. It's always better to use **const** or **let**.

File: javascript/javascript1/week2/index4.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+function weatherWear(num){
+    if(num>=0 && num<15)
+     return clothes1
+     else if (num>15)
+      return clothes2
+      else
+        return clothes3
+}
+
+console.log(weatherWear(24))
```
Comment (esteficodes) [inline javascript/javascript1/week2/index4.js:17]: The logic here is perfect! But check out your vars. Your function is suggesting shorts and a t-shirt for a temperature of 0 degrees ! 🥶🥶🥶

File: javascript/javascript1/week2/index5.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    } 
+  
+}
+
+addStudentToClass("A")
+
+function getNumberOfStudents() {
+  return console.log(class07Students.length)
+}
+getNumberOfStudents()
```
Comment (esteficodes) [inline javascript/javascript1/week2/index5.js:32]: Ok! 👍

File: javascript/javascript1/week2/index6.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+addCandy("sweet", 10);
+addCandy("toffee", 5);
+addCandy("chocolate", 25)
+
+if (boughtCandyPrices.length > 0) {
+    if (canBuyMoreCandy(amountToSpend)) 
+        console.log("You can buy more, so please do!");
+    else 
+        console.log("Enough candy for you!");    
+}
```
Comment (esteficodes) [inline javascript/javascript1/week2/index6.js:51]: Sugar rush! 🍭🍭🍬

**Review by esteficodes — APPROVED** [review]
Good job! After modifying some minor things in the comments, you may merge your homework. 👍 😀

File: javascript/javascript1/week2/index2.js (hunk 1 above)
Comment (chunmeishui) [reply javascript/javascript1/week2/index2.js:26]: i think you should use || in stead of && in line 4.

Comment (esteficodes) [reply javascript/javascript1/week2/index2.js:26]: @chunmei the vars fullname1 and fullname2 respond to name and surname respectively, so you need to use && to return the full name. Or do you see something that I'm missing? 🤔

**Pull request merged**